build/docker/package.arm64.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -euxo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+VERSION=1.26.2
```

Comment:
You need to add this file to `SetAllVersions.cs`, otherwise it won't be updated when we change the version number

Reply:
Oh right, I forgot that...

Reply:
done in 3768c28